module/VuFind/src/VuFind/Controller/AbstractBase.php

```diff
@@ -726,6 +726,9 @@ protected function setFollowupUrlToReferer(bool $allowCurrentUrl = true)
             return;
         }
 
+        // Clear previously stored lightboxParent.
+        $this->followup()->retrieveAndClear('lightboxParent');
```

Comment:
You could just use `->clear` here since you're not using the retrieved value.

Reply:
fixed